packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersForm.tsx

```diff
@@ -113,74 +112,61 @@ export const InviteMembersForm = (props: InviteMembersFormProps) => {
+          direction='col'
+          gap={2}
+          sx={{ width: '100%' }}
+        >
+          <Text localizationKey={localizationKeys('formFieldLabel__emailAddresses')} />
+
+          <Text
+            localizationKey={localizationKeys('formFieldInputPlaceholder__emailAddresses')}
+            colorScheme='neutral'
+            sx={t => ({ fontSize: t.fontSizes.$xs })}
+          />
 
-      <Form.Root onSubmit={onSubmit}>
-        <Form.ControlRow elementId={emailAddressField.id}>
-          <Flex
-            direction='col'
-            gap={2}
+          <TagInput
+            {...restEmailAddressProps}
+            autoFocus
+            validate={isEmail}
             sx={{ width: '100%' }}
-          >
-            <Text localizationKey={localizationKeys('formFieldLabel__emailAddresses')} />
-
-            <Text
-              localizationKey={localizationKeys('formFieldInputPlaceholder__emailAddresses')}
-              colorScheme='neutral'
-              sx={t => ({ fontSize: t.fontSizes.$xs })}
-            />
-
-            <TagInput
-              {...restEmailAddressProps}
-              autoFocus
-              validate={isEmail}
-              sx={{ width: '100%' }}
-              validateUnsubmittedEmail={validateUnsubmittedEmail}
-            />
-          </Flex>
-        </Form.ControlRow>
-        <Form.ControlRow elementId={roleField.id}>
-          <Flex
-            direction='col'
-            gap={2}
-          >
-            <Text localizationKey={localizationKeys('formFieldLabel__role')} />
-            {/*// @ts-expect-error */}
-            <Select
-              elementId='role'
-              {...roleField.props}
-              onChange={option => roleField.setValue(option.value)}
-            >
-              <SelectButton sx={t => ({ width: t.sizes.$48, justifyContent: 'space-between', display: 'flex' })}>
-                {roleField.props.options?.find(o => o.value === roleField.value)?.label}
-              </SelectButton>
-              <SelectOptionList sx={t => ({ minWidth: t.sizes.$48 })} />
-            </Select>
-          </Flex>
-        </Form.ControlRow>
-        <FormButtonContainer>
-          <Form.SubmitButton
-            block={false}
-            isDisabled={!canSubmit}
-            localizationKey={localizationKeys('organizationProfile.invitePage.formButtonPrimary__continue')}
+            validateUnsubmittedEmail={validateUnsubmittedEmail}
           />
-          <Form.ResetButton
-            localizationKey={resetButtonLabel || localizationKeys('userProfile.formButtonReset')}
-            block={false}
-            onClick={onReset}
-          />
-        </FormButtonContainer>
-      </Form.Root>
-    </>
+        </Flex>
+      </Form.ControlRow>
+      <Form.ControlRow elementId={roleField.id}>
+        <Flex
+          direction='col'
+          gap={2}
+        >
+          <Text localizationKey={localizationKeys('formFieldLabel__role')} />
+          {/* @ts-expect-error */}
```

Comment:
❓ Why are we expecting an error? Can we address it or add a description?

Reply:
Added the comments, stuff like that is what i am to solve when refactoring our forms